diff -ur php-5.6.0beta2-mysqlicopy/ext/mysqli/mysqli_priv.h php-5.6.0beta2/ext/mysqli/mysqli_priv.h
--- php-5.6.0beta2-mysqlicopy/ext/mysqli/mysqli_priv.h	2014-05-01 14:42:03.000000000 +0000
+++ php-5.6.0beta2/ext/mysqli/mysqli_priv.h	2014-05-03 06:48:48.721183453 +0000
@@ -118,7 +118,7 @@
 #else
 /* libmysql */
 #define MYSQLI_ASYNC	 	0
-#define MYSQLI_STORE_RESULT_OFS	0
+#define MYSQLI_STORE_RESULT_COPY_DATA 0
 #endif
 
 /* for mysqli_fetch_assoc */
