diff -ur php-5.6.0RC2-phpdbgman/sapi/phpdbg/Makefile.frag php-5.6.0RC2/sapi/phpdbg/Makefile.frag
--- php-5.6.0RC2-phpdbgman/sapi/phpdbg/Makefile.frag	2014-07-02 00:23:22.000000000 +0000
+++ php-5.6.0RC2/sapi/phpdbg/Makefile.frag	2014-07-05 09:00:25.530189744 +0000
@@ -28,7 +28,7 @@
 	@$(INSTALL) -m 0755 $(BUILD_BINARY) $(INSTALL_ROOT)$(bindir)/$(program_prefix)phpdbg$(program_suffix)$(EXEEXT)
 	@echo "Installing phpdbg man page:       $(INSTALL_ROOT)$(mandir)/man1/"
 	@$(mkinstalldirs) $(INSTALL_ROOT)$(mandir)/man1
-	@$(INSTALL_DATA) sapi/phpdbg/phpdbg.1 $(INSTALL_ROOT)$(mandir)/man1/$(program_prefix)phpdbg$(program_suffix).1
+	@$(INSTALL_DATA) $(srcdir)/phpdbg.1 $(INSTALL_ROOT)$(mandir)/man1/$(program_prefix)phpdbg$(program_suffix).1
 
 clean-phpdbg:
 	@echo "Cleaning phpdbg object files ..."
